autodeploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
```

Comment:
Maybe also use `set -v` or `set -x` (see http://wiki.bash-hackers.org/scripting/debuggingtips#use_shell_debug_output for the difference) for better logs on Travis.